app/controllers/foods_controller.rb

```diff
@@ -52,19 +50,20 @@ def destroy
     @food.destroy
 
     respond_to do |format|
-      format.html { redirect_to foods_url, notice: "Food was successfully destroyed." }
+      format.html { redirect_to foods_url, notice: 'Food was successfully destroyed.' }
       format.json { head :no_content }
     end
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_food
-      @food = Food.find(params[:id])
-    end
 
-    # Only allow a list of trusted parameters through.
-    def food_params
-      params.require(:food).permit(:name, :measure_unit, :price, :quantity, :user_id)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_food
+    @food = Food.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def food_params
+    params.require(:food).permit(:name, :measure_unit, :price, :quantity, :user_id)
```

Comment:
`user_id` is not passed as a `params` when you create the food and it won't be able to find the user to create the food. 

- [ ] Kindly update the to creating the food instead use the `current_user` to find the logged-in user and create the food.